### ISSUE TITLE
Question: display a custom capture message when a player has found a flag (no story mode)?

### ISSUE THREAD

**Issue by ClaudioWayne** [moloch--/RootTheBox#630]
Is there a way to display a custom capture message when a player has found a flag if the story mode is disabled? 
I try to show the players several solutions that lead to the goal of the flag, so that they can compare their solution with the alternatives and can always revisit them when they have found the flag.

Thanks
Claudio

**Comment by eljeffeg** [comment]
I think it just triggers if in story mode if I recall,  but you can change the graphic to be more "did you know".  And I think if there is no text, it will skip a "cut scene".

**Comment by ClaudioWayne** [comment]
Exactly. That works for me. You can close my two questions if u want. I just need to figure out how SWAT and Federal Reserve work, but I guess it's a little buggy atm.
Thx for your work.
